src/test/java/it/pagopa/selfcare/pagopa/backoffice/service/CreditorInstitutionServiceTest.java

```diff
@@ -0,0 +1,253 @@
+
+        assertDoesNotThrow(() -> service.deleteCreditorInstitutionStationRelationship("12345678900", "00000000000_01"));
+    }
+
+    @Test
+    void deleteCreditorInstitutionStationRelationship_ko() {
+        FeignException feignException = mock(FeignException.InternalServerError.class);
+        doThrow(feignException).when(apiConfigClient).deleteCreditorInstitutionStationRelationship(anyString(), anyString());
+
+        assertThrows(FeignException.class, () -> service.deleteCreditorInstitutionStationRelationship("12345678900", "00000000000_01"));
+    }
+
+    @Test
+    void createCreditorInstitution_ok() throws IOException {
+        when(apiConfigClient.createCreditorInstitution(any(CreditorInstitutionDetails.class)))
+                .thenReturn(TestUtil.fileToObject("response/apiconfig/post_creditor_institution_ok.json", CreditorInstitutionDetails.class));
+
+        CreditorInstitutionDto dto = TestUtil.fileToObject("request/post_creditor_institution.json", CreditorInstitutionDto.class);
+        CreditorInstitutionDetailsResource result = service.createCreditorInstitution(dto);
+
+        assertNotNull(result);
+    }
+
+    @Test
+    void createCreditorInstitution_ko() throws IOException {
+        FeignException feignException = mock(FeignException.InternalServerError.class);
+        when(apiConfigClient.createCreditorInstitution(any(CreditorInstitutionDetails.class))).thenThrow(feignException);
+
+        CreditorInstitutionDto dto = TestUtil.fileToObject("request/post_creditor_institution.json", CreditorInstitutionDto.class);
+        assertThrows(FeignException.class, () -> service.createCreditorInstitution(dto));
+    }
+
+    @Test
+    void createCIAndBroker_ok() throws IOException {
+        when(apiConfigClient.createCreditorInstitution(any(CreditorInstitutionDetails.class)))
+                .thenReturn(TestUtil.fileToObject("response/apiconfig/post_creditor_institution_ok.json", CreditorInstitutionDetails.class));
+        when(apiConfigClient.createBroker(any(BrokerDetails.class)))
+                .thenReturn(new BrokerDetails());
+
+        CreditorInstitutionAndBrokerDto dto = TestUtil.fileToObject("request/post_creditor_institution_and_broker.json", CreditorInstitutionAndBrokerDto.class);
+        CreditorInstitutionDetailsResource result = service.createCIAndBroker(dto);
+
+        assertNotNull(result);
+    }
+
+    @Test
+    void createCIAndBroker_ko() throws IOException {
+        FeignException feignException = mock(FeignException.InternalServerError.class);
+        when(apiConfigClient.createCreditorInstitution(any(CreditorInstitutionDetails.class))).thenThrow(feignException);
+
+        CreditorInstitutionAndBrokerDto dto = TestUtil.fileToObject("request/post_creditor_institution_and_broker.json", CreditorInstitutionAndBrokerDto.class);
+        assertThrows(FeignException.class, () -> service.createCIAndBroker(dto));
+        verify(apiConfigClient, times(0)).createBroker(any(BrokerDetails.class));
```

Comment:
I want to give you some advice. Next time you can use `mockito.captor` like [this](https://github.com/pagopa/pagopa-selfcare-ms-backoffice-backend/blob/cb11bbe2edb4d6af9cba43b2b5da22b002418228/src/test/java/it/pagopa/selfcare/pagopa/backoffice/scheduler/CiBrokerExtractionSchedulerTest.java#L67)